src/main/java/core/basesyntax/service/implemantation/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.implemantation;
+
+import core.basesyntax.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReaderImpl implements FileReader {
+    @Override
+    public List<String> readDate(String fileNameToRead) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(
+                new java.io.FileReader(fileNameToRead))) {
```

Comment:
just add import java.io.FileReader;
```suggestion
                new FileReader(fileNameToRead))) {
```